<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Lato</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>e-Ledger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/img/ledger_small.png</normaloff>:/img/img/ledger_small.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loginHeader">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>96</y>
      <width>201</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
font-size: 35px;</string>
    </property>
    <property name="text">
     <string>USER LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>323</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click here to login.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#loginButton {
	background-color: #32D736;
	border-radius: 10px;
	color: #fff;
	font-size: 20px;
	padding: 8px;
}

#loginButton:hover {
	background-color: #34E038;
}


</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="usernameGroup">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>184</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="loginUsernameIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
font-size: 20px;
qproperty-alignment: AlignCenter;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>loginUsernameInput</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginUsernameInput">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>0</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your valid username.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginUsernameInput {
	padding: 10px;
	background-color: #fff;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
	font-size: 20px;
	color: #374957;
}

#loginUsernameInput:focus {
	background-color: #E3E8F0;
}</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="passwordGroup">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>254</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="loginPasswordInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your valid password.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginPasswordInput {
	padding: 10px;
	font-size: 20px;
	color: #374957;
}

#loginPasswordInput:focus {
	background-color:#E3E8F0;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginPasswordIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
font-size: 20px;
qproperty-alignment: AlignCenter;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>loginPasswordInput</cstring>
     </property>
    </widget>
    <widget class="QPushButton" name="show_hide_pwd_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Click here to view/hide password.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
font-size: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="noAccountGroup">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>141</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="createAccount">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Click here to create a new account.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#createAccount {
	font-size: 15px;
	background-color: transparent;
	color: #32D736;
}

#createAccount:hover {
	color: #3EF642;
}</string>
     </property>
     <property name="text">
      <string>Create one</string>
     </property>
    </widget>
    <widget class="QLabel" name="noAccountMessage">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
background-color: transparent;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Don't have an account?</string>
     </property>
     <property name="buddy">
      <cstring>createAccount</cstring>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="forgot_password">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>363</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Click here to reset your password.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#forgot_password {
	font-size: 15px;
	background-color: transparent;
	color: #E3E8F0;
}

#forgot_password:hover {
	color: #FFF;
}</string>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_button">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>20</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Help</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>loginUsernameInput</tabstop>
  <tabstop>loginPasswordInput</tabstop>
  <tabstop>show_hide_pwd_btn</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>forgot_password</tabstop>
  <tabstop>createAccount</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
